functions/python-sqs-trigger-hello-world/README.md

```diff
@@ -0,0 +1,71 @@
+2023-08-18 15:24:06.178	DEBUG - Function Triggered: / source=core
+2023-08-18 15:23:36.166 DEBUG - 127.0.0.1 - - [18/Aug/2023:13:23:36 +0000] "POST / HTTP/1.1" 200 19 "-" "ScalewayServerlessOrchestrator/1.0" source=user stream=stdout
+2023-08-18 15:23:36.166	DEBUG - Error source=user stream=stdout
+2023-08-18 15:23:36.166	DEBUG - Received body: {"message": 3} source=user stream=stdout
+2023-08-18 15:23:36.160	DEBUG - Function Triggered: / source=core
+2023-08-18 15:23:06.149	DEBUG - 127.0.0.1 - - [18/Aug/2023:13:23:06 +0000] "POST / HTTP/1.1" 200 19 "-" "ScalewayServerlessOrchestrator/1.0" source=user stream=stdout
+2023-08-18 15:23:06.149	DEBUG - Error source=user stream=stdout
+2023-08-18 15:23:06.149	DEBUG - Received body: {"message": 3} source=user stream=stdout
+2023-08-18 15:23:06.148	DEBUG - 127.0.0.1 - - [18/Aug/2023:13:23:05 +0000] "POST / HTTP/1.1" 200 5 "-" "ScalewayServerlessOrchestrator/1.0" source=user stream=stdout
+2023-08-18 15:23:06.147	DEBUG - Success source=user stream=stdout
+2023-08-18 15:23:06.147	DEBUG - Received body: {"message": 2} source=user stream=stdout
+2023-08-18 15:23:06.063	DEBUG - Function Triggered: / source=core
+2023-08-18 15:22:36.152	DEBUG - 127.0.0.1 - - [18/Aug/2023:13:22:36 +0000] "POST / HTTP/1.1" 200 5 "-" "ScalewayServerlessOrchestrator/1.0" source=user stream=stdout
+2023-08-18 15:22:36.152	DEBUG - Success source=user stream=stdout
+2023-08-18 15:22:36.152	DEBUG - Received body: {"message": 5} source=user stream=stdout
+2023-08-18 15:22:36.147	DEBUG - 127.0.0.1 - - [18/Aug/2023:13:22:35 +0000] "POST / HTTP/1.1" 200 19 "-" "ScalewayServerlessOrchestrator/1.0" source=user stream=stdout
+2023-08-18 15:22:36.147	DEBUG - Error source=user stream=stdout
+2023-08-18 15:22:36.147	DEBUG - Received body: {"message": 2} source=user stream=stdout
+2023-08-18 15:22:36.048	DEBUG - 127.0.0.1 - - [18/Aug/2023:13:22:35 +0000] "POST / HTTP/1.1" 200 19 "-" "ScalewayServerlessOrchestrator/1.0" source=user stream=stdout
+2023-08-18 15:22:36.047	DEBUG - Error source=user stream=stdout
+2023-08-18 15:22:36.047 DEBUG - Received body: {"message": 3} source=user stream=stdout
+2023-08-18 15:22:35.852	DEBUG - 127.0.0.1 - - [18/Aug/2023:13:22:35 +0000] "POST / HTTP/1.1" 200 5 "-" "ScalewayServerlessOrchestrator/1.0" source=user stream=stdout
+2023-08-18 15:22:35.852	DEBUG - Success source=user stream=stdout
+2023-08-18 15:22:35.851	DEBUG - Received body: {"message": 1} source=user stream=stdout
+2023-08-18 15:22:35.849	DEBUG - 127.0.0.1 - - [18/Aug/2023:13:22:35 +0000] "POST / HTTP/1.1" 200 5 "-" "ScalewayServerlessOrchestrator/1.0" source=user stream=stdout
+2023-08-18 15:22:35.848	DEBUG - Success source=user stream=stdout
+2023-08-18 15:22:35.848	DEBUG - Received body: {"message": 4} source=user stream=stdout
+2023-08-18 15:22:14.449	DEBUG - [2023-08-18 13:22:14 +0000] [12] [INFO] Booting worker with pid: 12 source=user stream=stderr
+2023-08-18 15:22:12.849	DEBUG - Function Triggered: / source=core
+```
```

Comment:
Same comment re. verbosity, maybe strip the dates and times

Reply:
done